packages/dcos-integration-test/extra/conftest.py

```diff
@@ -21,6 +21,31 @@
 LOGIN_PW = os.getenv('DCOS_LOGIN_PW')
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers', 'order(n): mark test to run in Nth order.'
+        'Negative numbers represent position from the end.'
+        'EG: 1, 2, 3, ... unordered ... -3, -2, -1')
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """Reorders test using order mark
+    """
+    first = []
+    unordered = []
+    last = []
+    for item in items:
+        if not hasattr(item.obj, 'order'):
```

Comment:
just reversing using either reverse or making the key func negate i[0] will cause us to go [3, 2, 1, 0, -1, -2, -3]. I _think_ that is the ordering we want (Destructor order should exactly reverse constructor order based on programming language history to date).

if the `[1, 2, 3, 0, -1, -2, -3]` is wanted, that's just another key function. If we can guarantee size, we could do bucketing more simply, but I don't think that is the right order for this sort of thing.